spring-rabbit/src/test/java/org/springframework/amqp/rabbit/core/AsyncRabbitTemplateTests.java

```diff
@@ -0,0 +1,298 @@
+/*
+ * Copyright 2014 the original author or authors.
```

Comment:
Something wrong with your IDE config?
Or is that just a copy/paste artefact?

Will fix on merge